app/controllers/comments_controller.rb

```diff
@@ -59,6 +64,10 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
+
```

Comment:
nitpik: please be mindful of unnecessary white spaces

Reply:
changed!